WSL/wsl-config.md

```diff
@@ -291,3 +291,37 @@ These options are available in Insider Build 17713 and later.
 |:----|:----|:----|:----|
 | enabled | boolean | `true` | Setting this key will determine whether WSL will support launching Windows processes. |
 | appendWindowsPath | boolean | `true` | Setting this key will determine whether WSL will add Windows path elements to the $PATH environment variable. |
+
+#### user
+
+Section label: `[user]`
+
+These options are available in Build 18980 and later.
+
+| key | value | default | notes|
+|:----|:----|:----|:----|
+| default | string | The initial username created on first run | Setting this key specifies which user to run as when first starting a WSL session. |
+
+## Configure global options with .wslconfig
+
+> **Available in Windows Build 19041 and later**
+
+You can configure global WSL options by placing a `.wslconfig` file into the root directory of your users folder: `C:\Users\<yourUserName>\.wslconfig`. This file can contain the following options:
+
+### WSL 2 Settings
+
+These settings affect the VM that powers any WSL 2 distribution. 
+
+| key | value | default | notes|
+|:----|:----|:----|:----|
+| kernel | string | The Microsoft built kernel provided inbox | An absolute Windows path to a custom Linux kernel. |
+| memory | size | 70% of your total memory on Windows | How much memory to assign to the WSL 2 VM. |
```

Comment:
@benhillis can you double check the default value here?

Reply:
80%